golos.vesting/golos.vesting.cpp

```diff
@@ -359,9 +360,11 @@ void vesting::close(name owner, symbol symbol) {
 }
 
 void vesting::notify_balance_change(name owner, asset diff) {
+    tables::vesting_table table_vesting(_self, _self.value);
+    auto notify = table_vesting.find(diff.symbol.code().raw());
```

Comment:
I suppose we need a test for this functionality. I would like to see in the test case with different precision for token and vesting.

Reply:
Added to description of #314